src/components/Card.tsx

```diff
@@ -74,7 +76,8 @@ const cardContainer = (
   nospace?: boolean,
   useAppStyles?: boolean,
   paddingType?: 'small' | 'normal',
-  spinner?: string
+  spinner?: string,
+  bigCard?: boolean
```

Comment:
We have `useAppStyles`, can we use them for this `only-app` cards? We should modify that prop

Reply:
I was able to get rid of the bigCard property entirely